types/connect-busboy/connect-busboy-tests.ts

```diff
@@ -1,6 +1,13 @@
-import connectBusboy = require('connect-busboy');
+import * as connectBusboy from 'connect-busboy';
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import and call/construct this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```